scripts/Packages/Package_DFN_QFN/size_definitions/qfn-64_9x9.yaml

```diff
@@ -151,6 +151,59 @@ QFN-64-1EP_9x9mm_P0.5mm_EP6x6mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-64-1EP_9x9mm_P0.5mm_EP7.15x7.15mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/229321fa.pdf (page 27)'
```

Comment:
please use the #page=27 feature for the url to point the webbrowser to the correct page.